lib/autobot.ts

```diff
@@ -41,7 +34,7 @@ type PRHookNames = keyof Hooks["pr"];
 
 export interface Hooks {
   [ExecutionScope.App]: {
-    onStart: SyncBailHook<[Application], Autobot>;
+    onStart: SyncHook<[Application]>;
```

Comment:
I don't actually have great use-case for the app level plugins yet 😬. I wrote a test for it though, and this being a bail hook didn't make a lot of sense. If it really needs to _bail_ that likely means an error has occurred and it should just throw an exception instead.

Reply:
On second thought, I'll probably use it for analytics